projects/uitdatabank/reference/entry.json

```
@@ -1554,6 +1554,43 @@
           "$ref": "#/components/parameters/eventId"
         }
       ],
+      "get": {
+        "summary": "contributors - get",
+        "operationId": "event-contributors-get",
+        "responses": {
+          "200": {
+            "description": "An array of contributors.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "../models/common-contributors.json"
+                }
+              }
+            }
+          },
+          "401": {
+            "$ref": "#/components/responses/Unauthorized"
+          },
+          "403": {
+            "$ref": "#/components/responses/Forbidden"
+          },
+          "404": {
+            "$ref": "#/components/responses/NotFound"
+          }
+        },
+        "tags": [
+          "Events"
+        ],
+        "description": "Returns a JSON array of contributors, meaning those that have edit rights on the event.",
```

```suggestion
        "description": "Returns a JSON array of contributors, meaning users that have edit rights on the event.",
```
(Just a minor suggestion - should also be changed in the others if you agree)